tutorials/RichMenu.md

```diff
@@ -0,0 +1,78 @@
+While {@tutorial RichDisplay} allows you to create fully configurable paginated embeds, {@link RichMenu} allows you to define menus using a similar interface.
+
+Unlike {@tutorial RichDisplay}, {@link RichMenu} doesn't allow to define how pages will look and instead of calling {@link RichDisplay.addPage} that allows to customize the template (if provided), the user is presented with the {@link RichMenu.addOption} method, which only requires a `name` and a `body` and handles the organization of the options automatically.
```

Comment:
>doesn't allow to define how pages will

could use better wording

Reply:
manages a page's content and layout automatically?